zoo/base/assets/js/project_detail.js

```diff
@@ -67,7 +67,7 @@ $(document).ready(() => {
     }
   })
 
-  $('.ui.project-actions.dropdown').dropdown({
+  $('.ui.fade.dropdown').dropdown({
```

Comment:
The `.ui.project-actions.dropdown` is still valid, and it's more specific 🤔

Reply:
Or am I missing something? 😅

Reply:
I changed it to `fade` so that I don't have to create a specific `service_detail.js` because of that. I think it doesn't have to be that much specific so that we can use it in components on the page.

I added `fade` class to the `project-actions` dropdown, so it should work. This is just generalization so that I don't have to repeat this code specifically for every rendered dropdown.

Reply:
What I mean is that I would have to repeat:

```js
  $('.ui.XXX.dropdown').dropdown({
    action: 'hide',
    position: 'right',
    transition: 'fade up'
  })
```

Three times as the service detail page contains 3 dropdowns now. And would have to create `service_detail.js` specific for service detail view.

Reply:
Hmm... got your point. But I don't think any of this has value towards libraries, which share this js file, and we'll end up adding more logic to the `service_detail` page, which is the one that needs all the fancy stuff.

It also makes more sense from the point of keeping a consistent structuring, at least in my opinion, feel free to let me know otherwise.

But I'm ok with leaving it as it is for now and move it more in the future 🙂 I'll leave it up to you

Reply:
OK, I would leave as it is for now. I think dropdowns make sense in libraries as well as there is already one dropdown (the delete/edit actions), so it should be here even after we create `service_detail.js` specific for Services (and I definitely agree we should at some point - but IMO there is no need for now).